### PR TITLE
add clean adb forward list and lock file after checking the mtbf  pro…

### PR DESCRIPTION
**Pull request by ShakoHo**
add clean adb forward list and lock file after checking the mtbf  process is not running
@zapion , @ypwalter r?

### CODE REVIEW

File: moz_minions/kevin.py [hunk 1]
```diff
@@ -177,6 +179,31 @@ def check_process_exist(self):
...
+        output = proc.communicate()[0]
+        if proc.stdin:
+            proc.stdin.close()
+        if proc.stdout:
+            proc.stdout.close()
+        if proc.stderr:
+            proc.stderr.close()
+        return output
+
+    def remove_adb_forward_list(self, serial_no):
```
Comment (zapion) [inline moz_minions/kevin.py:198]: how do you think to apply askeing's adb wrapper?
https://github.com/askeing/b2g-util-python/blob/master/b2g_util/util/adb_helper.py
Or you may keep impl., just a comment.

Comment (ShakoHo) [inline moz_minions/kevin.py:198]: I've considered this option, but it seems we don't have forward argument support in adb helper...but I still can go that way ....but that will take a little bit longer to deliver these code~
how do you think?

**Comment by zapion** [comment]
@ShakoHo 
see if it is suitable to our case
https://github.com/askeing/b2g-util-python/pull/8

**Comment by ShakoHo** [comment]
sure, if that PR is merged, we can leverage the b2g-util-python

**Comment by zapion** [comment]
adb forward feature merged in https://github.com/askeing/b2g-util-python/pull/9

**Pull request closed without merging**